full-stack-tests/core/src/common/IModelHubUserMgr.ts

```diff
@@ -5,13 +5,19 @@
 
 import { BeEvent } from "@bentley/bentleyjs-core";
 import { FrontendAuthorizationClient } from "@bentley/frontend-authorization-client";
-import { AccessToken, UserInfo } from "@bentley/itwin-client";
+import { AccessToken } from "@bentley/itwin-client";
 import { getAccessTokenFromBackend } from "@bentley/oidc-signin-tool/lib/frontend";
 
 export class IModelHubUserMgr implements FrontendAuthorizationClient {
   private _token: AccessToken | undefined;
+  private _expiresAt?: Date; // TODO: Find way to set this
 
-  public constructor(_userInfo: UserInfo | undefined, private _userCredentials: any) {
+  public constructor(private _userCredentials: any) {
+  }
+
+  public isExpired(token?: AccessToken ): boolean {
+    token = token ?? this._token;
+    return !(token === this._token && this._expiresAt !== undefined && this._expiresAt.getTime() - Date.now() >= 1 * 60 * 1000);
```

Comment:
@calebmshafer I previously added in `_expiresAt` for the token expiration, but we don't have a straight forward way of getting an expiration. After looking a little further it seems this and all the almost identical classes (TestIModelHubOidcAuthorizationClient & nativeapp's IModelHubUserMgr) are all used exclusively in full stack tests - so I'm thinking expiration tracking isn't even needed. Do you agree?

Reply:
I'd have to agree. I don't think it's needed. We'll want to run the integration tests to be sure though.